CONTRIBUTING.md

```diff
@@ -207,23 +207,35 @@ npm run test-jasmine -- --help
 npm run test-jasmine -- --info
 ```
 
-### Image pixel comparison tests
-
-Image pixel comparison tests are run in a docker container. For more
-information on how to run them locally, please refer to [image test
-README](https://github.com/plotly/plotly.js/blob/master/test/image/README.md).
-
-Running the test locally outputs the generated png images in `build/test_images/` and the png diffs in `build/test_images_diff/` (two git-ignored directories).
-
-To view the image pixel comparison test results, run
-
+### Draft new baseline
+Install fonts and tools
+```sh
+# install required fonts (if missing) on ubuntu
+sudo cp -r .circleci/fonts/ /usr/share/ && sudo fc-cache -f
+# upgrade pip (if needed)
+python3 -m pip install --upgrade pip
```

Comment:
To be clear re: my earlier comments - here the instructions are for local use, so using the longer `python3 -m pip` syntax is reasonable as a way to indicate to users that it's important to do this with Python 3 if nothing else :)